crates/primitives/res/genesis/dev.json

```diff
@@ -0,0 +1,18 @@
+{
+  "nonce": "0x00",
+  "timestamp": "0x6159af19",
+  "extraData": "0x5365706f6c69612c20417468656e732c204174746963612c2047726565636521",
+  "gasLimit": "0x1c9c380",
+  "difficulty": "0x20000",
+  "mixHash": "0x0000000000000000000000000000000000000000000000000000000000000000",
+  "coinbase": "0x0000000000000000000000000000000000000000",
+  "stateRoot": "0x5eb6e371a698b8d68f665192350ffcecbbbf322916f4b51bd79bb6887da3f494",
+  "alloc": {
+    "0x6Be02d1d3665660d22FF9624b7BE0551ee1Ac91b": {
+      "balance": "0xD3C21BCECCEDA1000000"
+    }
+  },
+  "number": "0x0",
+  "gasUsed": "0x0",
+  "parentHash": "0x0000000000000000000000000000000000000000000000000000000000000000"
+}
```

Comment:
for example, where is this from? we would want users to be able to spend the coins in `alloc`, and while I was able to find the private key online `0x99b3c12287537e38c90a9219d4cb074a89a16e9cdb20bf85728ebd97c343e342`, we would need a better way to generate this